app/app/urls.py

```diff
@@ -14,6 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import include, path
 
-urlpatterns = [path("admin/", admin.site.urls)]
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("accounts/", include("django.contrib.auth.urls")),
```

Comment:
per the AC "Login page is created as the base landing page for the system."

Ie, localhost:8000/  (or wherever the server is running) should lead to the login page (either through redirect, or this just being the base, or however else you can think of it.

Reply:
Done wait for commit

Reply:
Addressed in b3276c6